src/ripple/core/impl/DatabaseCon.cpp

```diff
@@ -38,9 +40,134 @@ setup_DatabaseCon(Config const& c)
         Throw<std::runtime_error>("database_path must be set.");
     }
 
+    if (!setup.globalPragma)
+    {
+        setup.globalPragma = [&c, &j]() {
+            auto const& sqlite = c.section("sqlite");
+            auto result = std::make_unique<std::vector<std::string>>();
+            result->reserve(3);
+
+            // defaults
+            std::string safety_level;
+            std::string journal_mode = "wal";
+            std::string synchronous = "normal";
+            std::string temp_store = "file";
+            bool showRiskWarning = false;
+
+            if (set(safety_level, "safety_level", sqlite))
+            {
+                showRiskWarning = boost::iequals(safety_level, "low");
+                if (showRiskWarning)
```

Comment:
I'd rewrite this to make the condition `boost::iequals(safety_level, "low")` and then setting `showRiskWarning` in the if statement. That way it's clearer that we're running that code if the `safety_level` is low, it's only incidentally related to `showRiskWarning`.

Reply:
That's how I had it the first time, but I changed it to match the pattern on the other items. But it's not quite the same pattern, and it does make sense to do it your way here.